index.html

```diff
@@ -11,11 +11,13 @@
 
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
let's move this down a with the other stylesheets.
nit: put the `rel` attribute first to match the others